app/forms/school_cohort_form.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class SchoolCohortForm
+  include ActiveModel::Model
+
+  attr_accessor :estimated_mentor_count, :estimated_teacher_count
+
+  validates :estimated_mentor_count, presence: { message: "Enter your expected number of mentors" }
+  validates :estimated_teacher_count, presence: { message: "Enter your expected number of ECTs" }
```

Comment:
Do we want 'more or equal than 0' validation here?

Reply:
sure